go/codegen/experimental/codegen_normal_stmt.go

```diff
@@ -55,7 +55,7 @@ func generateNormalStmtStep(stmt string, stepIndex int, session *pb.Session) (st
 	filler := &normalStmtFiller{
 		StepIndex:  stepIndex,
 		DataSource: session.DbConnStr,
-		Stmt:       stmt,
+		Stmt:       replaceNewLineRuneAndTrimSpace(stmt),
```

Comment:
Is this a bug fix? It doesn't look like "adding more testcases"?

Reply:
Yes. It is a bug fix.